.eslintrc.js

```diff
@@ -43,5 +41,9 @@ module.exports = {
     "no-console": ["off"],
     // Prefer default exports
     "import/prefer-default-export": ["off"],
-  },
-}
+    "import/no-extraneous-dependencies": [
+      "error",
+      { devDependencies: ["**/__stories__/*", "**/__tests__/*", "/test/*"] }
+    ]
```

Comment:
added this rule to fix linting problems in the storybook files, the rest of this is just prettier churn